qiskit_nature/algorithms/pes_samplers/bopes_sampler.py

```diff
@@ -61,14 +65,16 @@ def __init__(
                 all previous points will be used for bootstrapping.
             extrapolator: Extrapolator objects that define space/window
                            and method to extrapolate variational parameters.
+            gss: (DEPRECATED) Old name for the solver when it only supported GroundStateEigensolver.
```

Comment:
Since the prior version said GroundStateSolver as the type perhaps this should say that here. Maybe in practice the GroundStateEigensolver subclass was all that we had and that was supported in practice but it said the type was a GroundStateSolver

Reply:
I agree. I guess in theory an `AdaptVQE` could also have been used here, but I don't think I ever tested that

Reply:
Indeed, this was a typo, thanks